modules/new_serverpod_auth/serverpod_auth_idp/serverpod_auth_idp_server/analysis_options.yaml

```
@@ -21,11 +1,6 @@
 
 analyzer:
   exclude:
     - lib/src/generated/**
-    - test/integration/test_tools/serverpod_test_tools.dart
-
-# For more information about the core and recommended set of lints, see
-# https://dart.dev/go/core-lints
-
-# For additional information about configuring this file, see
-# https://dart.dev/guides/language/analysis-options
+    - test/test_tools/serverpod_test_tools.dart
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)